datetime/time.rs

```diff
@@ -20,9 +20,11 @@ use plib::PROJECT_NAME;
 
 #[derive(Parser)]
 #[command(
-    version,
     about = gettext("time - time a simple command or give resource usage"),
-    help_template = gettext("{about-with-newline}\nUsage: {usage}\n\nArguments:\n{positionals}\n\nOptions:\n{options}")
+    help_template = gettext("{about}\n\nUsage: {usage}\n\nArguments:\n{positionals}\n\nOptions:\n{options}"),
+    version = env!("CARGO_PKG_VERSION"),
```

Comment:
1) is this `version =` setting required of all posixutils utilities?  Surely the default value is correct and wanted?

2) is a help template _for each posixutils utility_ required, in order to support translation?

Reply:
1. No. Fixed.
2. Yes, but the help template may differ